packages/nestjs/src/decorators.ts

```diff
@@ -19,12 +19,12 @@ export const SentryCron = (monitorSlug: string, monitorConfig?: MonitorConfig):
           try {
             result = originalMethod.apply(this, args);
           } catch (e) {
-            captureException(e);
+            captureException(e, { mechanism: { handled: false, type: 'auto.cron.nestjs' } });
```

Comment:
`cron` is not a [listed category](https://develop.sentry.dev/sdk/telemetry/traces/span-operations/), which is a bit suboptimal. The two alternatives are:

- `function` given we wrap some kind of cron-executed function
- omitting the category, since it's not required: `auto.nestjs.sentry_cron` 

@AbhiPrasad thoughts?

Reply:
let's do `auto.function.X`